packages/paste-website/src/pages/utilities/screen-reader-only/index.mdx

```diff
@@ -0,0 +1,212 @@
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/screen-reader-only/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Screen Reader Only"
+  categoryRoute={SidebarCategoryRoutes.UTILITIES}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/utilities/screen-reader-only"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/utilities-screen-reader-only--default"
+  data={props.data.allPasteUtility.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+### About Screen Reader Only
+
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+When wrapped with the Screen Reader Only component, content is visually hidden but still available to assistive technology.
+
+This can be useful when context or relationships between UI controls are _visually_ clear, but in isolation to an assistive technology user, they may not be. By being able to visually hide some content, you are able to provide additional clarifying context that may only be clear to a sighted user.
+
+
+## Examples
+
+### Providing headings to sectioning content
+
+All sectioning elements, such as Cards, should contain a heading to provide a clear document outline and form of navigation to assistive technology users. By using the Screen Reader Only component you can provide a visually hidden heading to give additional context and heirarchy.
+
+<LivePreview scope={{Card, ScreenReaderOnly, Heading, Paragraph, Button, Anchor, Flex}} language="jsx">
+  {`<Card>
+  <ScreenReaderOnly as="div">
+    <Heading as="h2" variant="heading20">Flex</Heading>
+  </ScreenReaderOnly>
+  <Paragraph>
+    Deploy an omnichannel contact center within minutes and customize every element of the experience, including the agent desktop, channels, interaction routing, and reporting using common web frameworks like React.js and REST APIs.
+  </Paragraph>
+  <Flex vAlignContent="center">
+    <Flex marginRight="space40">
+      <Button variant="primary">
+        Launch Flex
+      </Button>
+    </Flex>
+    <Anchor href="#">
+      View all projects
+    </Anchor>
+  </Flex>
+</Card>`}
+</LivePreview>
+
+### Repetitive buttons and hidden headers
+
+Each column in a table is required to have a column header. Sometimes those column headers can be visually hidden from sighted users if desired. To do so use the Screen Reader Only component to hide the cell content for that column header.
+
+Another common use case is for providing context to repetitive actions. Visually the "Buy" button is associated with the phone number via the row highlighting, but to a screen reader user the "Buy" action might be unclear. To provide additional clarification we can add _what_ you would be buying as visually hidden text to the button.
```

Comment:
I would write this as:

Another common use case is for providing correct context to repetitive actions. In the example below, the "Buy" button is clearly visually associated with the phone number via row highlighting. However, to a screen reader user, this association may not be as clear. To provide additional clarification we can add _what_ you would be buying (Buy <associated number>) as visually hidden text to the button. This way, we can keep the same level of context for all users.